### ISSUE TITLE
Lack of animation for the Construction Kit

### ISSUE THREAD

**Issue by AWxxx** [Unvanquished/Unvanquished#3149]
Lack of animation for the Construction Kit in [Unvanquished impacts the overall gameplay experience. I believe that incorporating a toned-down chainsaw animation would significantly enhance this aspect of the game.

**Comment by VReaperV** [comment]
I don't really get what chainsaw animation has to do with that. Is it supposed to make something move on the ckit? Or create some particles/hit effects?

**Comment by AWxxx** [comment]
Yeah, it could be a welding effect, like flashes and such—something that doesn’t strain the eyes. So I was thinking about an effect similar to what a saw has, but in a "smaller version"

**Comment by Gireen** [comment]
Ohh you mean when the ckit is used for repairing? Or also when starting construction?

**Comment by AWxxx** [comment]
Repair  or construction
[Example](https://www.youtube.com/watch?v=Jt_rlVzU4FYl)
